packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -85,20 +96,20 @@ export default class UIController {
     this.TAB_NAMES_DISABLED = 'disabled'
   }
 
-  init ({ api, store, uiOptions } = {}) {
+  init ({ api, store } = {}) {
     if (!api) {
       throw new Error('API object is required for a UI controller initialization')
     }
     if (!store) {
       throw new Error('Vuex store is required for a UI controller initialization')
     }
-    if (!uiOptions) {
-      throw new Error('UI options are required for a UI controller initialization')
-    }
 
     this._api = api
     this._store = store
-    this._uiOptions = uiOptions
+    this._uiOptions = this._api.settings.getUiOptions()
+
+    // Inject HTML code of a plugin. Should go in reverse order.
```

Comment:
what does "should go in reverse order" mean?

Reply:
I think this code is obsolete. It was referring to the order of initialization of HTML elements and their values. It's not relevant any more. I will remove it.

Reply:
thanks.